src/components/CampaignForm.tsx

```diff
@@ -0,0 +1,203 @@
+import { Form } from 'react-final-form';
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import {  Autocomplete, TextField } from 'mui-rff';
+import { Avatar, Box, Button, MenuItem, Typography } from '@material-ui/core';
+import { Grid, GridSize } from '@material-ui/core';
+import { FormattedMessage as Msg, useIntl } from 'react-intl';
+import { useEffect, useState } from 'react';
+
+import getSearchDrawerResults from '../fetching/getSearchDrawerResults';
```

Comment:
I think since this is getting used here as well, I think you could rename this function in this PR to `getPeopleSearchResults` and update that in the `SearchDrawer` component as well.